src/components/drag-and-drop/custom-drag-layer/custom-drag-layer.js

```diff
@@ -62,6 +62,10 @@ class CustomDragLayer extends React.Component {
     };
   }
 
+  getKlassName = (props) => {
+    return `custom-drag-layer ${props.className || ''}`;
```

Comment:
We use the [classNames](https://github.com/JedWatson/classnames) package in Carbon to handle this:

```
classNames('custom-drag-layer', props.className);
```